sdk/resources/arm-resources/tsconfig.json

```diff
@@ -3,12 +3,13 @@
     "module": "es6",
     "moduleResolution": "node",
     "strict": true,
-    "target": "es5",
+    "target": "es6",
     "sourceMap": true,
     "declarationMap": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
+    "preserveConstEnums": true,
     "lib": ["es6", "dom"],
     "declaration": true,
     "outDir": "./esm",
```

Comment:
@sarangan12 
```suggestion
    "outDir": "./dist-esm",
```

Reply:
@deyaaeldeen @sarangan12 if the `outDir` is changed to `./dist-esm`, please also change the `api-extractor.json`.

Reply:
@deyaaeldeen @sarangan12 Please also change the `module` in `package.json`

Reply:
@dw511214992 thanks for the investigation, this will be fixed in code gen in a future release.